### ISSUE TITLE
Please can you add logits extraction functionalities ? 

### ISSUE THREAD

**Issue by Elimane0800** [ollama/ollama#7951]
Is there any way to extract ollama models logits please ? It'll be very helpful for those who work in AI research. If there's already a func like this with ollama please expose it to us so that we can continue using ollama.

**Comment by rick-github** [comment]
https://github.com/ollama/ollama/issues/2415

**Comment by rick-github** [comment]
closing as dupe #2415

**Comment by jmorganca** [comment]
Closing for https://github.com/ollama/ollama/issues/2415
